src/pages/index.js

```diff
@@ -1,11 +1,13 @@
 import HomeNavbar from "../components/homeNavbar";
+import HomeHeader from "../components/homeHeader";
 import React from "react";
 
 export default function Index() {
   return (
 
     <div>
       <HomeNavbar />
+      <HomeHeader />
```

Comment:
You can add Grid here and do `<Grid` xs={12} md={6}>` for your container that only needs to take half of the screen. Also add margin from the navbar.